fundamentals/minimal-apis/samples/MinApiTestsSample/MinApiTests/UnitTests/TodoInMemoryTests.cs

```diff
@@ -14,7 +14,7 @@ public async Task GetTodoReturnsNotFoundIfNotExists()
         await using var context = new MockDb().CreateDbContext();
 
         // Act
-        var notFoundResult = (NotFound)await TodoEndpointsV1.GetTodo(404, context);
```

Comment:
@fiyazbinhasan is there a reason you're looking for `ID=404`? I found it a bit confusing because you're expecting a 404 response.

Reply:
no specific reason. you can use any id that is not available in the db